h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2423,24 +2456,11 @@ else if (gaussian.equals(_parms._family) && Link.identity.equals(_parms._link))
           }
         }
         double[] zvalues = MemoryManager.malloc8d(_state.activeData().fullN() + 1);
-        Cholesky chol = _chol;
-        DataInfo activeData = _state.activeData();
-        if (_parms._standardize) { // compute non-standardized t(X)%*%W%*%X
-          double[] beta_nostd = activeData.denormalizeBeta(beta);
-          DataInfo.TransformType transform = activeData._predictor_transform;
-          activeData.setPredictorTransform(DataInfo.TransformType.NONE);
-          Gram g = new GLMIterationTask(_job._key, activeData, new GLMWeightsFun(_parms), beta_nostd).doAll(activeData._adaptedFrame)._gram;
-          activeData.setPredictorTransform(transform); // just in case, restore the transform
-          g.mul(_parms._obj_reg);
```

Comment:
This parms._obj_reg = 1/Number of row of data.  It is like an averaging effect.  However, when I am playing with the formulae, no averaging is needed.  Plus I will be doing things like matrix and vector operatons.  If I keep the average constant, I need to add it to the vectors that I put into the formula.  This will only slow things down and make the code hard to understand.

Reply:
Ok, thank for your explanation!